src/recipes/fadeTransition.ts

```diff
@@ -27,6 +28,9 @@ const fadeTransitionRecipe = defineSlotRecipe({
         enterActive: { transition: `opacity {durations.distractionFreeTyping} ease 0ms` },
         exitActive: { transition: `opacity {durations.slow} ease 0ms` },
       },
+      nodeOpacity: {
+        exitActive: { transition: `opacity {durations.nodeOpacity} ease-out` },
```

Comment:
Should this also set `enter: { opacity: 1 },` to completely avoid the `enter` animation?

Reply:
Yes, I've added `enter: { opacity: 1 }` to avoid the enter animation.